client/src/components/ClubDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
What's the purpose of this component? It isn't used anywhere? It looks like it'd serve a similar purpose as `ClubPage` right?

Reply:
Took it out.